client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -178,13 +242,20 @@ class ActivityLogDay extends Component {
 }
 
 export default connect(
-	( state, { tsEndOfSiteDay } ) => {
+	( state, { tsEndOfSiteDay, siteId } ) => {
 		const now = Date.now();
 		return {
 			isToday: now <= tsEndOfSiteDay && tsEndOfSiteDay - DAY_IN_MILLISECONDS <= now,
+			requestedRewind: getRequestedRewind( state, siteId ),
 		};
 	},
 	{
 		recordTracksEvent: recordTracksEventAction,
+		rewindRequestDismiss: rewindRequestDismissAction,
+		trackRewindCancel: siteId =>
+			withAnalyticsAction(
```

Comment:
any particular reason for the rename of this from `withAnalytics()`?

Reply:
I just followed what was done in this and other files in Activity Log project: imported actions are renamed to {action}Action.
I actually don't see much worth in doing this, but 1) I thought it was something that was agreed upon so I went with it and 2) since it's done in almost all cases I wanted to preserve consistency and did the same.

Reply:
It’s definitely not anything agreed upon. The only reason I think people rename those actions is because `eslint` yells at them for shadowing names.

We can get around it in other ways which are also prevalent in Calypso code:
 - choose names that don’t conflict when designing selectors and UI: for example `chooseSitePlan()` action and `choosePlan()` name in the component
 - refer to the actions via `this.props.actionName()` instead of restructuring them out

I tend to prefer the first solution because it encourages us to decouple the UI from the state system. That is, the interface for the specific UI element is usually much more generic than we are thinking when we are building it. A plan item may just need `onClick` or `selectPlan` while our actions tend to carry more contextual information which is provided and scaffolded in `connect()`

In general though it’s not assume any patterns are deliberate or any existing code should be an example. 😉